library/insights_config.py

```diff
@@ -0,0 +1,49 @@
+ANSIBLE_METADATA = {
+     'metadata_version': '1.1',
+     'status': ['preview'],
+     'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: insights_config
+short_description: This module handles initial configuration of the insights client on install
+description:
+  - Supply values for various configuration options that you would like to use. On install
+  this module will add those values to the insights-client.conf file prior to registering.
+version_added: "3.0"
+options:
+  insights_name:
+    description:
+    - For now this is just 'insights-client', but that could change in the future if the
+    product name changes.
+    required: true
```

Comment:
Same as above, let's make this optional, only needed for backwards compatibility.